production/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+services:
+  app:
+    build: .
+    ports:
+      - "0.0.0.0:9001:9001"
+    environment:
+      DB_HOST: pg
+      DB_USER: postgres
+      DB_USER_PASS: ""
+      RACK_ENV: production
+    depends_on:
+      - pg
+    command: curl --retry 10000 --retry-delay 10000 http://something.com
```

Comment:
DB_USER_PASS is ok, as our compose file is for testing production mode, not to run it on a production machine. I hope if people will use it for their production they would make more sane choices for passwords :)